index.html

```diff
@@ -121,6 +121,25 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ Hau Trong Phan________  -->
+        <div class="card">
+          <p class="name">Hau Trong Phan</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/hauphanlvc" target="_blank">hauphanlvc</a>
+          </p>
+          <p class="about">
+            Hi, I'm just a person who want to know how to contribute a project, and become a professional full stack developer. 
+          </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://free-for.dev/#/" target="_blank" title="First Resource">Get all free services </a>
+              </li>
+            </ul>
+          </div>
```

Comment:
In this resources section, providing 3 helpful online resources is a good practice! It seems you have provided 1 resource link, which is fine. But, adding two more is recommended. 

You could add any resource link that the fellow developers could make use of !